Xamarin.Forms.Platform.iOS/CollectionView/CarouselViewController.cs

```diff
@@ -56,7 +84,10 @@ public override void ViewDidLayoutSubviews()
 				_size = CollectionView.Bounds.Size;
 				BoundsSizeChanged();
 			}
-			
+
+			if (Carousel.Loop)
```

Comment:
maybe make a convenience property for this. `ShouldLoop => if (Carousel?.Loop == true && _carouselViewLoopManager != null)`
```suggestion
			if (Carousel?.Loop == true && _carouselViewLoopManager != null)
```